lib/index.ts

```diff
@@ -31,6 +31,7 @@ export function debug(...messages: string[]) {
 }
 
 export interface MavenOptions extends legacyPlugin.BaseInspectOptions {
+  'print-graph'?: boolean;
```

Comment:
Is there a way to keep this consistent with the camel-casing of the other options? I'm suspecting that the args parser of the CLI already overloads the options object with `print-graph` and `printGraph`.

Reply:
If that's possible, then it's a great idea. I was not aware of any overloads since before.

Reply:
I did some investigation. It's not possible to use camel case out of the box, since it's just a subset of the options that gets transformed. But it's still possible to fix it.

To accomplish this, we need to add "print-graph" to the [following list](https://github.com/snyk/cli/blob/main/src/cli/args.ts#L225). And add 'print-graph' to the set of possible values, defined [here](https://github.com/snyk/cli/blob/main/src/lib/types.ts#L234).

The drawback is that it will impact other parts of the code base as well since the old key will be removed in runtime at [this place](https://github.com/snyk/cli/blob/main/src/cli/args.ts#L234).

Reply:
Oof, ok. 😅 Thanks for doing such a deep investigation @dekelund. It might be best for the maintainers of this plugin to make a decision on whether such a deep change is necessary to keep the casing in line with other options.